tests/Model/StubsContainer.php

```diff
@@ -125,8 +125,9 @@ public function getFunction($name, $sourceFilePath = null, $shouldSuitCurrentPhp
         }
         if (!empty($functions)) {
             return array_pop($functions);
+        } else {
+            throw new RuntimeException("Could not get function {$name} from reflection, possibly unsupported PHP_VERSION in getenv");
```

Comment:
Minor: Curly braces are unnecessary here. Other than that PR is good to go.

Reply:
Thanks, noted. Will try to address within one of the next PRs.